split_tunnel_test.go

```diff
@@ -0,0 +1,256 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSplitTunnelIncludeHost(t *testing.T) {
+	setup()
+	defer teardown()
+
+	handler := func(w http.ResponseWriter, r *http.Request) {
+		assert.Equal(t, http.MethodGet, r.Method, "Expected method 'GET', got %s", r.Method)
+		w.Header().Set("content-type", "application/json")
+		fmt.Fprintf(w, `{
+			"success": true,
+			"errors": [],
+			"messages": [],
+			"result": [
+				{
+					"host": "*.example.com",
+					"description": "default"
+			 }
+			]
+		}
+		`)
+	}
+
+	want := []SplitTunnel{{
+		Host: "*.example.com",
+		Description: "default",
+	}}
+
+	mux.HandleFunc("/accounts/"+testAccountID+"/devices/policy/include", handler)
+
+	actual, err := client.SplitTunnelInclude(context.Background(), testAccountID)
+
+	if assert.NoError(t, err) {
+		assert.Equal(t, want, actual)
+	}
+}
+
+func TestSplitTunnelIncludeAddress(t *testing.T) {
+	setup()
+	defer teardown()
+
+	handler := func(w http.ResponseWriter, r *http.Request) {
+		assert.Equal(t, http.MethodGet, r.Method, "Expected method 'GET', got %s", r.Method)
+		w.Header().Set("content-type", "application/json")
+		fmt.Fprintf(w, `{
+			"success": true,
+			"errors": [],
+			"messages": [],
+			"result": [
+				{
+					"address": "10.0.0.0/8",
+					"description": "default address"
+			 }
+			]
+		}
+		`)
+	}
+
+	want := []SplitTunnel{{
+		Address: "10.0.0.0/8",
+		Description: "default address",
+	}}
+
+	mux.HandleFunc("/accounts/"+testAccountID+"/devices/policy/include", handler)
+
+	actual, err := client.SplitTunnelInclude(context.Background(), testAccountID)
+
+	if assert.NoError(t, err) {
+		assert.Equal(t, want, actual)
+	}
+}
+
+func TestUpdateSplitTunnelInclude(t *testing.T) {
+	setup()
+	defer teardown()
+
+	handler := func(w http.ResponseWriter, r *http.Request) {
+		assert.Equal(t, http.MethodPut, r.Method, "Expected method 'PUT', got %s", r.Method)
+		w.Header().Set("content-type", "application/json")
+		fmt.Fprintf(w, `{
+			"success": true,
+			"errors": [],
+			"messages": [],
+			"result": [
+				{
+					"address": "10.0.0.0/8",
+					"description": "default address"
+			 },
+			 {
+				  "address": "100.64.0.0/10",
```

Comment:
can we update this to use the IPs from [rfc5737](https://datatracker.ietf.org/doc/html/rfc5737#section-3)?